docroot/profiles/sdss/sdss_profile/sdss_profile.post_update.php

```diff
@@ -5,106 +5,3 @@
  * sdss_profile.post_update
  */
 
-use Drupal\block_content\Entity\BlockContent;
-use Drupal\block\Entity\Block;
-use Drupal\Core\Serialization\Yaml;
-
-/**
- * Disable the core search module.
- */
-function sdss_profile_post_update_8200() {
```

Comment:
Make sure to add in a `hook_removed_post_updates` when you remove these.

Reply:
Added 👍